src/main/java/de/tum/in/www1/artemis/domain/ShortAnswerSubmittedText.java

```diff
@@ -104,9 +105,15 @@ public void setSubmittedAnswer(ShortAnswerSubmittedAnswer shortAnswerSubmittedAn
     }
     // jhipster-needle-entity-add-getters-setters - JHipster will add getters and setters here, do not remove
 
-    // TODO FDE: check if text input is correct needs to improve
-    public boolean isSubmittedTextCorrect(String submittedText, String solution){
-        return submittedText.equalsIgnoreCase(solution);
+    public boolean isSubmittedTextCorrect(String submittedText, String solution) {
```

Comment:
please add some Java Doc documentation, otherwise no one will understand the purpose of this method ;-)

Reply:
yeah, you are right. thank you for your feedback! :)